### PR TITLE
Setup release workflow

### PR DESCRIPTION
**Pull request by nils-werner-sonarsource**

### CODE REVIEW

File: .github/workflows/release.yml [hunk 1]
```diff
@@ -0,0 +1,79 @@
...
+          CIRRUS_TOKEN: ${{ secrets.CIRRUS_TOKEN }}
+          PATH_PREFIX: ${{ secrets.BINARIES_PATH_PREFIX }}
+          GITHUB_TOKEN: ${{ secrets.RELEASE_GITHUB_TOKEN }}
+          RELEASE_SSH_USER: ${{ secrets.RELEASE_SSH_USER }}
+          RELEASE_SSH_KEY: ${{ secrets.RELEASE_SSH_KEY }}
+          SLACK_API_TOKEN: ${{secrets.SLACK_API_TOKEN }}
+        uses: SonarSource/gh-action_release/main@master
+        with:
+          publish_to_binaries: true
+          attach_artifacts_to_github_release: true
```
Comment (tomverin) [inline .github/workflows/release.yml:28]: I've just updated the documentation, this option is not available anymore

File: .github/workflows/release.yml [hunk 2]
```diff
@@ -0,0 +1,79 @@
...
+          distribute: true
+          distribute_target: false
+          run_rules_cov: false
```
Comment (tomverin) [inline .github/workflows/release.yml:31]: Same here, you should remove this option

File: .github/workflows/release.yml [hunk 3]
```diff
@@ -0,0 +1,79 @@
...
+          slack_channel: team-lang-phpy
```
Comment (tomverin) [inline .github/workflows/release.yml:32]: are you sure about the y at the end?

**Review by tomverin — CHANGES_REQUESTED** [review]

File: .github/workflows/release.yml (hunk 3 above)
Comment (nils-werner-sonarsource) [reply .github/workflows/release.yml:32]: Yeah. We merged PHP and Python bubble and call it PHPy.

**Review by tomverin — APPROVED** [review]

**Pull request merged**